docs/content/docs/build/containers.md

```diff
@@ -3,28 +3,139 @@ title: Containers
+    },
+    dynamicObjectTypes: [ SharedCell, SharedString ],
+}
+const { container, containerServices} = await client.getContainer(/*service config*/, schema);
+```
 
 ### Attaching a container
 
+Once the `createContainer` call resolves the container is connected to the service (attached) and available to collaborative. 
+
+We are currently working on an advanced scenario where containers can be created locally before persisting them to the server. The primary use case for this is if a client want's to draft the initial state of a container before other collaborators enter.
+
 ### Deleting a container
 
-## Lifecycle
+Deleting a container is a server specific policy and you should refer to the documentation associated with the specific Fluid service you are using.
+
+## Interaction & lifecycle
+
+### connected/disconnected
+
+The container exposes current connected state and emits connected and disconnected events to notify the caller if the underlying connection is disrupted. Fluid will by default attempt to reconnect in case of lost/intermittent connectivity.
+
+```typescript
+const connected = container.connected();
+
+container.on("disconnected", () => {
+    // handle disconnected
+    // prevent the user from editing to avoid data loss
+});
+
+container.on("connected", () => {
+    // handle connected
+    // enable editing if disabled
+});
+```
 
-### connected/disconnected/disposed
+### dispose
+
+The container exposes functionality that allow cleanup of the container object. Calling dispose will remove any server connections and clean up registered events. Once a container is dispose it cannot be re-hydrated. Connecting to the same container requires another `getContainer(...)` call.
+
+```typescript
+container.dispose();
+
+const disposed = container.disposed();
+
+container.on("disposed", () => {
+    // handle event cleanup to prevent memory leaks
+});
+```
+
+### initialObjects
+
+`initialObjects` are the base set of `LoadableObjects` in a container. They exist for the lifetime of the container and are guaranteed to be available and loadable. They are defined via the container schema and match the signature on the schema. 
+
+For more on `initialObjects` see [Data modeling](data-modeling.md)
+
+### create
+
+The container also exposes a create function that allows dynamic `LoadableObject` creation.
+
+For more on dynamic object creation see [Data modeling](data-modeling.md)
 
 ## Patterns for managing container lifecycle
 
-### Create should be separate from load
+### Create/load separation
+
+When creating and loading a container our basic guidance is to separate the two flows. This provides a cleaner separation of responsibilities within the code it self. From a scenario perspective it manifests as the creator going through a explicit creation process that results in a redirect to a new page who's sole responsibility is to load the container. All users will load the container through this subsequent flow.
```

Comment:
I'm not sure I really like this section since it's a bit to prescriptive. I do like that it highlights creation though so probably worth keeping for now.